katalog/cilium/README.md

```diff
@@ -0,0 +1,60 @@
+# Cilium
+
+<!-- <KFD-DOCS> -->
+
+**Cilium** is an open source, cloud native solution for providing, securing, and observing network connectivity between
+workloads, fueled by the revolutionary Kernel technology eBPF.
+
+> For more information about Cilium refer to [cilium documentation][cilium-documentation]
+
+The deployment of Cilium consists of a DaemonSet running on all nodes, and a operator Deployment.
+Additionally, we deploy hubble component as an observability tool on the network connections between pods in the cluster.
+
+> ⚠️ please notice that the Cilium package is for cluster with less than 200 nodes.
+
+## Image repository and tag
+
+- cilium images:
+  - `registry.sighup.io/fury/cilium/cilium:v1.13.1`
+  - `registry.sighup.io/fury/cilium/operator-generic:v1.13.1`
+  - `registry.sighup.io/fury/cilium/hubble-ui-backend:v0.10.0`
+  - `registry.sighup.io/fury/cilium/hubble-ui:v0.10.0`
+  - `registry.sighup.io/fury/cilium/hubble-relay:v1.13.1`
+
+## Requirements
+
+- Kubernetes >= `1.23.X`.
+- Kustomize >= `v3.5.3`.
+- [prometheus-operator from KFD monitoring module][prometheus-operator]
+- [cert-manager from KFD ingress module][cert-manager]
+
+## Configuration
+
+The Cilium package is deployed with the following configuration:
+
+- TBD
```

Comment:
Cilium uses this config by default https://docs.cilium.io/en/v1.13/network/concepts/ipam/cluster-pool/ that is not leveraging the CIDR from kubeadm for pods. For the SVC CIDR is still used the one in the cluster, since cilium is not managing that